src/main/java/calculator/DelimiterParser.java

```diff
@@ -0,0 +1,54 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DelimiterParser {
+
+    private static final String CUSTOM_DELIMITER_PREFIX = "//"; // 커스텀 구분자 접두사
+    private static final String CUSTOM_DELIMITER_SUFFIX = "\\n"; // 커스텀 구분자 접미사
+
+    public String[] parseNumbers(String input) {
+        List<String> delimiterList = new ArrayList<>(Arrays.asList(",", ":"));
```

Comment:
```suggestion
    public String[] parseNumbers(String input) {
        List<String> delimiters = new ArrayList<>(Arrays.asList(",", ":"));
```

`List`라는 것은 자료형만으로 충분히 파악이 가능해요. 이름을 바꿔 보는 건 어떨까요??